services/frontend-service/src/ui/NavigationBar/NavigationBar.scss

```diff
@@ -13,6 +13,20 @@
   }
 
   .mdc-drawer__content {
-    height: 100%;
+    height: 95%;
+  }
+
+  .mdc-list-item-indicator {
+    background-color: transparent;
+    width: 5%;
+  }
+
+  .mdc-list-item-indicator--activated {
+    background-color: orange;
+    transition: background-color 0.2s ease-in-out;
+  }
+
+  .mdc-list-item-container {
+    display: flex
```

Comment:
Please add:
`height: 100%;`
so that the container can also take the full size of the parent (the three navigation buttons fill up the space of the drawer instead of just the height of the content which is just the icons.

Reply:
Okay, fixed!